docs/source/getting_started/faq.rst

```diff
@@ -21,6 +21,17 @@ Development
 -  `How can you
    contribute? <https://github.com/jupyterlab/jupyterlab/blob/master/CONTRIBUTING.md>`__
 -  :ref:`How can you extend or customize JupyterLab? <user_extensions>`
+-  In the classic Notebook, `I could use custom Javascript outputed by a cell to programatically
+   control the Notebook <https://stackoverflow.com/a/32769976/907060>`__. Can I do the same thing in JupyterLab?
+
+   JupyterLab was built to support a wide variety of extensibility, including dynamic behavior based on notebook
+   outputs.To access this extensability, you should write a custom JuptyerLab extension. If you would
```

Comment:
missing space after period, before "To access"

Reply:
Thanks!